app/models/cfe/base_result.rb

```diff
@@ -152,7 +156,7 @@ def vehicle_assessed_amount
     ################################################################
 
     def pensioner_capital_disregard
-      if version_4? || version_5? || version_empty?
+      if version_4? || version_5? || version_6? || version_empty?
```

Comment:
not sure of context here but I assume there is a theoretical version 3, and below, so would `version >= 4` version be neater, with maybe a rewrite of version method
```
def version
  result_hash.fetch(:version, "1").sub(/empty/i, "4").to_i
end
```

but as **nit** as test suite explosion and anxiety could ensue 😄